pkg/quarkus/v1alpha/scaffolds/api.go

```diff
@@ -62,19 +62,19 @@ func (s *apiScaffolder) Scaffold() error {
 	var createAPITemplates []machinery.Builder
 	createAPITemplates = append(createAPITemplates,
 		&model.Model{
-			Package:   util.ReverseDomain(s.config.GetDomain()),
+			Package:   util.ReverseDomain(util.SanitizeDomain(s.config.GetDomain())),
```

Comment:
Not relevant to the current PR but maybe it'd make sense to compute these (and the class name) only once.